.eslintrc.json

```diff
@@ -1,10 +1,12 @@
 {
     "extends": "airbnb",
+    "parser": "babel-eslint",
     "parserOptions": { 
       "ecmaVersion": 6,
       "sourceType": "module",
       "ecmaFeatures": {
-        "jsx": true
+        "jsx": true,
+        "experimentalObjectRestSpread": true
```

Comment:
why is this property added?

Reply:
It helps resolve an eslint error concerning object spreading

Reply:
OK